lib/services/profiles-api-service.js

```diff
@@ -98,6 +98,17 @@ function profileApiServiceFactory(
                 }
             });
         }
+
+        if (query.macs && query.ips) {
```

Comment:
Need to combine the logic to handle the query pair `mac/ip` and `macs/ips` or we can completely remove the support for one of the query pair.

Recently I am studying the previous API proxy design, the previous code seems have some bug, so I'm acceptable to refactor this after we fully understand or fix the API proxy design.